src/mono/wasi/runtime/driver.c

```diff
@@ -63,7 +63,9 @@ int32_t monoeg_g_hasenv(const char *variable);
 void mono_free (void*);
 int32_t mini_parse_debug_option (const char *option);
 char *mono_method_get_full_name (MonoMethod *method);
+#ifndef INVARIANT_TIMEZONE
```

Comment:
Can we add a test for wasi/single file in this case?

Reply:
Created https://github.com/dotnet/runtime/issues/88127 for it

Reply:
There is an existing test for single file, it would need the extra case. Without the test we don't know if this works.